python/semantic_kernel/models/chat/chat_role.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
```

Comment:
A couple of new files in this PR are missing copyright header: `# Copyright (c) Microsoft. All rights reserved.`